tests/scripts/minikube.sh

```diff
@@ -81,6 +81,10 @@ case "${1:-}" in
     minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
     wait_for_ssh
     enable_roles_for_RBAC
+
+    # create a link so the default dataDirHostPath will work for this environment
+    minikube ssh "sudo mkdir /mnt/sda1/var/lib/rook;sudo ln -s /mnt/sda1/var/lib/rook /var/lib/rook"
```

Comment:
Shouldn't that be `mkdir -p ...`, or will `/mnt/sda1/var/lib` always pre-exist?

Reply:
yes, it's expected to pre-exist